packages/react-server/src/ReactFlightServer.js

```diff
@@ -521,14 +511,20 @@ export function resolveModelToJSON(
   }
 
   if (typeof value === 'symbol') {
+    const name = value.description;
```

Comment:
😲 TIL

Reply:
IIRC, it was actually added late or even later version of the spec.